src/Persistence/Initialization/VersionSeasonSix/Items/Weapons.cs

```diff
@@ -188,13 +188,13 @@ public override void Initialize()
         this.CreateWeapon(3, 10, 0, 22, 2, 4, true, "Dragon Spear", 92, 112, 140, 35, 85, 0, 0, 170, 60, 0, 0, 0, 2, 0, 0, 0, 0, 0);
         this.CreateWeapon(3, 11, 0, 22, 2, 4, false, "Beuroba", 147, 190, 226, 40, 90, 0, 0, 152, 25, 0, 0, 0, 2, 0, 1, 0, 0, 0);
 
-        this.CreateWeapon(4, 0, 1, 24, 2, 3, true, "Short Bow", 2, 3, 5, 30, 20, 0, 0, 20, 80, 0, 0, 0, 0, 1, 0, 0, 0, 0);
```

Comment:
We'll need an IConfigurationUpdatePlugIn for that to change existing configurations.

Reply:
the migration plugin should I keep in mind if someone changed the original values ​​to avoid replacing them?

Reply:
I think it's okay here to overwrite the values 😅

Reply:
In any case, I have seen that the updates have an "IsMandatory" flag, I will leave it at false so that it can be applied optionally, right?